src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_ListViewBase.cs

```diff
@@ -3618,6 +3618,41 @@ public async Task When_SelectionMode_Is_Multiple()
 			Assert.AreEqual(vsg.CurrentState?.Name, "MultiSelectEnabled");
 		}
 
+		[TestMethod]
```

Comment:
You'll need to put this test around `#if HAS_UNO` as it is using internal WinUI APIs that don't compile in the UWP sample on CI.

Reply:
done ✅

Reply:
done ✅